src/pages/volunteer-success.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const VolunteerSuccessPage = () => {
+    return (
+        <main>
+            <h1>Thank you for volunteering!</h1>
+            <p>You will receive a text message shortly.</p> <br />
+            <p>Please download OnFleet.</p>
```

Comment:
I think we should add a link to the app/website based on that:

| Device   | Link
| -------    | -----
| Apple     | [OnFleet App **AppStore** Link](https://apps.apple.com/us/app/onfleet-driver/id1084013403) |
| Android |  [OnFleet App **PlayStore** Link](https://play.google.com/store/apps/details?id=com.onfleet.driver.app) |
| Other     |  [OnFleet **Website** Link](https://onfleet.com/)